src/main/webapp/query-control.js

```diff
@@ -31,6 +31,11 @@ function clearPreviousResult() {
   document.getElementById('census-link').style.display = 'none';
   document.getElementById('toggle-data-btn').style.display = 'none';
   document.getElementById('map-options').style.display = 'none';
+  document.getElementById('yearly-total-chart').style.display = 'none';
```

Comment:
This function is now super long. Do we want to switch to having a list of ID names, and we do a forEach on the list to set the display to none? (Plus a couple other actions that are standalone, of course.)

Reply:
Great idea! Done!